### ISSUE TITLE
PHP-FPM fails to start after removing virtual server

### ISSUE THREAD

**Issue by aresxc** [virtualmin/virtualmin-nginx#56]
After removing a virtual server that uses PHP-FPM and Nginx, PHP-FPM fails to start. This is due to website pool not begin deleted after using `Virtualmin > Disable and Delete > Delete Virtual Server`.

`[08-Jun-2023 03:36:58] ERROR: [pool 1680217373282286] cannot get uid for user 'example.com'
[08-Jun-2023 03:36:58] ERROR: FPM initialization failed`

LEMP installation.
Ubuntu 22.04.2
Webmin 2.021
Virtualmin 7.7
PHP 8.1 (kinetic)

**Comment by iliajie** [comment]
Hello,

I didn't see this issue happening with Nginx. Nevertheless, we recently addressed it in a more general way, to make sure the PHP-FPM pool file is always removed.

Although, it's not clear exactly what's causing it. Do you have steps to reproduce this issue?

**Comment by aresxc** [comment]
Yes, I did it 3 times, with 3 Virtualmin installs, the 4 time I tried it didn't work.

Installation:
`./virtualmin-install.sh --bundle LEMP --minimal`

1. **Create Virtual Server:** example.com (Top-level server, default settings)
2. **Create Virtual Server:** test.example.com (Alias of example.com, default settings)
3. example.com > Disable and Delete > **Disable** Virtual Server > Also disable sub-servers? Yes
4. example.com > Disable and Delete > **Delete** Virtual Server > Yes, Delete It
